src/entity/units/darkPriest.ts

```diff
@@ -19,17 +19,17 @@ const unit: UnitSource = {
   },
   unitProps: {
     damage: 40,
-    attackRange: 264,
-    healthMax: 60,
+    attackRange: 300,
+    healthMax: 80,
     mana: 90,
     manaMax: 90,
     manaPerTurn: 30,
     manaCostToCast: 60,
   },
   spawnParams: {
-    probability: 20,
-    budgetCost: 10,
-    unavailableUntilLevelIndex: 8,
+    probability: 40,
+    budgetCost: 15,
+    unavailableUntilLevelIndex: 6,
```

Comment:
This is really early for a multi-cast enemy

Reply:
You think so? Maybe. I've never had any struggles with dark priests, and I figured increased enemy variety would be good

If you're saying it's too strong maybe we should increase the budget cost instead?

Again though: Midgame is too easy right now. Really the whole game, but especially midgame.